build-tools/android-toolchain/android-toolchain.targets

```diff
@@ -122,19 +122,19 @@
       DependsOnTargets="_SetMxeToolchainMakefileTimeToLastCommitTimestamp"
       Condition="$(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))"
       Inputs="..\..\external\mxe\Makefile"
-      Outputs="$(AndroidMxeFullPath)\bin\i686-w64-mingw32.static-gcc">
+      Outputs="$(AndroidMxeFullPath)\bin\i686-w64-mingw32.static-gcc;$(AndroidMxeFullPath)\bin\i686-w64-mingw32.static-cmake">
```

Comment:
Why the update to `Outputs`? The `make MXE_TARGETS` value is unchanged, so it won't actually build `i686-w64-mingw32.static-cmake`, will it?